pl_examples/domain_templates/semantic_segmentation.py

```diff
@@ -185,7 +182,7 @@ def main(hparams):
     # ------------------------
     # 1 INIT LIGHTNING MODEL
     # ------------------------
-    model = SegModel(hparams)
+    model = SegModel(**hparams)
```

Comment:
we shall be careful if the `hparams` contains any argument (key) not listed in the init it will crash